APMrover2/GCS_Mavlink.cpp

```diff
@@ -746,6 +746,12 @@ MAV_RESULT GCS_MAVLINK_Rover::handle_command_long_packet(const mavlink_command_l
                                               static_cast<int16_t>(packet.param3),
                                               packet.param4);
 
+    case MAV_CMD_DO_ENGINE_CONTROL:
```

Comment:
We have an ICE singleton; the handling of this message should be moved up to GCS_Common.cpp

Reply:
done